### ISSUE TITLE
Why feature"-"list when we have corelist? (naming convention: the dash)

### ISSUE THREAD

**Issue by thibaultduponchelle** [neilb/Perl-FeatureList#1]
Except this wondering, I love the idea! Thank you

**Comment by neilb** [comment]
Yeah, I did think about that, but somehow `corelist` seems ok, but `featurelist` doesn't. Maybe because "feature" is that bit longer?

I'm happy to be persuaded though, I only slightly leaned towards `feature-list`.

**Comment by thibaultduponchelle** [comment]
I don't know, maybe in the first place corelist should have be named "core-list" :grinning:

While looking at `corelist` I discovered the `--feature` option that *partially* implements Perl::FeatureList:
```bash
$ corelist --feature say

Data for 2017-09-22
feature "say" was first released with the perl v5.9.5 feature bundle
```

Probably you knew this and anyway it does not call the need for Perl::FeatureList into question... 😃 

I'm a bit confused that `feature-list` returns all features available to my current perl but `feature-list myversion` does not print the same at all (only the features *added* in my specific version).

I would maybe be more comfortable with:
`feature-list` return all features until my current perl version (current behaviour)
`feature-list a.b.c` returns all features until Perl "a.b.c" (not the same as today)
`feature-list a.b.c --strict` returns only features added in Perl "a.b.c" (--strict or another option like --exclusive or --added or whatever)

And If I could ask one more thing, on my computer, `corelist` has a man page 😄 

Sorry for the mess in this ticket, I'm really happy to see this new module arriving in the ecosystem, this is just great, the rest is just implementations details.
